README.md

```diff
@@ -59,71 +37,21 @@ Tauri currently supports development and distribution on the following platforms
 | :----------------------- | :-------------- |
 | Windows                  | 7 and above     |
 | macOS                    | 10.15 and above |
-| Linux                    | See below       |
-| iOS/iPadOS (coming soon) |                 |
-| Android (coming soon)    |                 |
-
-**Linux Support**
-
-For **developing** Tauri apps refer to the [Getting Started guide on tauri.app](https://tauri.app/v1/guides/getting-started/prerequisites#setting-up-linux).
-
-For **running** Tauri apps we support the below configurations (these are automatically added as dependencies for .deb and are bundled for AppImage so that your users don't need to manually install them):
-
-- Debian (Ubuntu 18.04 and above or equivalent) with the following packages installed:
-  - `libwebkit2gtk-4.1-0`, `libgtk-3-0`, `libayatana-appindicator3-1`<sup>1</sup>
-- Arch with the following packages installed:
-  - `webkit2gtk`, `gtk3`, `libayatana-appindicator`<sup>1</sup>
-- Fedora (latest 2 versions) with the following packages installed:
-  - `webkit2gtk3`, `gtk3`, `libappindicator-gtk3`<sup>1</sup>
-- Void with the following packages installed:
-  - `webkit2gtk`, `gtk+3`, `libappindicator`<sup>1</sup>
-
-<sup>1</sup> `appindicator` is only required if system trays are used
+| Linux                    | webkit2gtk 4.0 for Tauri v1 (for example Ubuntu 18.04). webkit2gtk 4.1 for Tauri v2 (for example Ubuntu 22.04). |
+| iOS/iPadOS (alpha)       | 9 and above     |
+| Android (alpha)          | 7 and above     |
 
-### Features
+## Getting Started
 
-- [x] Desktop Bundler (.app, .dmg, .deb, AppImage, .msi)
-- [x] Self Updater
-- [x] App Signing
-- [x] Native Notifications (toast)
-- [x] App Tray
-- [x] Core Plugin System
-- [x] Scoped Filesystem
-- [x] Sidecar
+If you are interested in making a tauri app, please visit the [documentation website](https://tauri.app).
 
-### Security Features
+The quickest way to get started is to install the [prerequisites](https://tauri.app/v1/guides/getting-started/prerequisites) for your system and create a new project with [`create-tauri-app`](https://github.com/tauri-apps/create-tauri-app/#usage). For example with `npm`:
 
-- [x] localhost-free (:fire:)
-- [x] custom protocol for secure mode
-- [x] Dynamic ahead of Time Compilation (dAoT) with functional tree-shaking
-- [x] functional Address Space Layout Randomization
-- [x] OTP salting of function names and messages at runtime
-- [x] CSP Injection
+```sh
+npm create tauri-app@latest
+```
```

Comment:
I would move this to be up as close to the top as possible to allow someone to immediately jump right in.